src/Serializer.php

```diff
@@ -61,9 +79,20 @@ public function serialize(\Closure $closure)
      */
     public function unserialize($serialized)
     {
-        /** @var SerializableClosure $unserialized */
         $unserialized = unserialize($serialized);
 
+        if ($this->signingKey) {
+            list($data, $signature) = $unserialized;
+            if ($signature !== $this->calculateSignature($data)) {
```

Comment:
Hmmm... I didn't know about that function probably because it's for PHP 5.6+. Any recommendations for something that needs to work on 5.4?